.github/workflows/deploy.yml

```
@@ -0,0 +1,78 @@
+name:  Spring Boot & Gradle CI/CD
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)